src/core/agent/services/signifyNotificationService.ts

```diff
@@ -234,23 +236,17 @@ class SignifyNotificationService extends AgentService {
       }
     }
 
-    if (notif.a.r === NotificationRoute.ExnIpexAgree) {
```

Comment:
Hmm, I think you removed this when fixing a merge conflict or something but this `ExnIpexAgree` is what you helped Duke cover yesterday.